mlflow/spark.py

```diff
@@ -815,30 +815,23 @@ def _load_pyfunc(path):
 
     :param path: Local filesystem path to the MLflow Model with the ``spark`` flavor.
     """
-    # NOTE: The getOrCreate() call below may change settings of the active session which we do not
-    # intend to do here. In particular, setting master to local[1] can break distributed clusters.
+    from mlflow.utils._spark_utils import (
+        _create_local_spark_session_for_loading_spark_model,
+        _get_active_spark_session,
+    )
+
+    # NOTE: The `_create_local_spark_session_for_loading_spark_model()` call below may change
+    # settings of the active session which we do not intend to do here.
+    # In particular, setting master to local[1] can break distributed clusters.
```

Comment:
I'd move this comment right above `_create_local_spark_session_for_loading_spark_model`.